packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -350,6 +351,61 @@ void main() {
     debugDefaultTargetPlatformOverride = null;
   });
 
+  testWidgets('cursor android golden', (WidgetTester tester) async {
+    final Widget widget = RepaintBoundary(
+      key: const ValueKey<int>(1),
+      child: CupertinoApp(
+        home: Center(
+          child: ConstrainedBox(
+            constraints: BoxConstraints.loose(const Size(400, 400)),
+            child: const CupertinoTextField(),
```

Comment:
You'd want to put the thing you want to golden test against in a repaintboundary. Otherwise you end up with a really big image like https://github.com/flutter/goldens/blob/master/packages/flutter/test/cupertino/text_field_cursor_test.0.0.png. 

You'd also want to isolate it from things like the debug banner etc that aren't specifically the scenario you're trying to test.